app/views/training/pages/_citation.html.slim

```diff
@@ -0,0 +1,10 @@
+.hf-citation
```

Comment:
All these classes are taken from elsewhere and copied in.
It is an opportunity for you to rewrite this properly and maintain the standard we've worked toward in the rest of the code.

Reply:
This file `app/views/training/pages/_citation.html.slim` could do with some consideration. Let's talk it over.